tfx/components/example_gen/import_example_gen/executor_test.py

```diff
@@ -140,6 +140,25 @@ def testDoWithProto(self):
         self.examples.get_string_custom_property(
             utils.PAYLOAD_FORMAT_PROPERTY_NAME))
 
+  def testDoWithSequenceExamples(self):
```

Comment:
create a sequence testdata

Reply:
yes, i added it under the testdata folder